code/controllers/subsystem/ticker.dm

```diff
@@ -645,7 +645,12 @@ SUBSYSTEM_DEF(ticker)
 		'sound/roundend/disappointed.ogg',
 		'sound/roundend/scrunglartiy.ogg',
 		'sound/roundend/petersondisappointed.ogg',
-		'sound/roundend/bully2.ogg'\
+		'sound/roundend/bully2.ogg',
+		'sound/roundend/ssethdisappointed.ogg',
```

Comment:
I don't think we want this one, at least not without editing the hippie part from it.

Reply:
yeah you might be right.
I could edit it to play a second of hava nagila instead of hippiestation, what do you think of that

Reply:
just remove this and think about editing it later.

Reply:
yeah ok